exporter/datadogexporter/internal/logs/translator.go

```diff
@@ -75,10 +75,14 @@ func Transform(lr plog.LogRecord, res pcommon.Resource) datadogV2.HTTPLogItem {
 
 	// we need to set log attributes as AdditionalProperties
 	// AdditionalProperties are treated as Datadog Log Attributes
+	var status string
 	lr.Attributes().Range(func(k string, v pcommon.Value) bool {
 		switch strings.ToLower(k) {
-		case "msg", "message":
+		// set of remapping are taken from Datadog Backend
+		case "msg", "message", "log":
 			l.Message = v.AsString()
+		case "status", "severity", "level", "syslog.severity":
```

Comment:
Where do these values come from?

Reply:
We discussed this offline, this comes from Datadog's backend code so we can't really link it here